### PR TITLE
[#48] Write testing instructions

### LINKED ISSUE #48: Configure testing and write instructions
So everyone can test GitHub application. Here is the application:

* https://github.com/apps/hint-man

It's created under my account but probably we need to create one under Kowainik. And also we probably need to create one more GitHub application for testing...

Also, probably we need to create an empty repository for testing. But this repository can be useful for both `hintman` and `hit`.

### PR DESCRIPTION
**Pull request by chshersh**
Resolves #48

For now, our backend just prints all incoming requests. So the only way to trigger event is to add an app to your repos. But we should create some empty test repo on Kowainik GitHub to test this app. Any ideas for the name? 🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,40 @@
...
+
+### How to run
+
+1. Run `smee` forwarder in a separate terminal window:
+    + `smee -u "https://smee.io/uTG0BCXnjq4DEff7" -p 8080`
+2. Run `backend`:
+    + `cabal new-run hintman -- -p 8080`
+3. Open GitHub application and press `Configure` button to add it to your account:
+    + https://github.com/apps/hint-man
+4. After adding access to your personal profie, choose `Select repositories`
```
Comment (vrom911) [inline README.md]:
```suggestion
4. After adding access to your personal profile, choose `Select repositories`
```

File: README.md [hunk 2]
```diff
@@ -6,4 +6,40 @@
...
+   repository access and it this application to any of your repositories. You
```
Comment (vrom911) [inline README.md]: Some word is missing here...

File: README.md [hunk 3]
```diff
@@ -6,4 +6,40 @@
...
+   adding and saving you should be able to see the following:
```
Comment (vrom911) [inline README.md]: You addidng and saving?

File: README.md [hunk 4]
```diff
@@ -6,4 +6,40 @@
...
+   + Incoming requests in running backend
```
Comment (vrom911) [inline README.md]:
```suggestion
   + Incoming requests in the running backend
```

File: README.md (hunk 3 above)
Comment (chshersh) [reply README.md]: Yeah, need to reword this sentence. I don't know what happened with it.

**Comment by chshersh** [comment]
@vrom911 Alternative name proposal: `hintman-target` 🙂

**Comment by vrom911** [comment]
I love it!

**Comment by chshersh** [comment]
Created the repo and added `hintman` to it:

* https://github.com/kowainik/hintman-target/settings/installations

So now every event in this repo can be used to test `hintman` 🙂

**Pull request merged**